clients/fossid-webapp/src/test/kotlin/FossIdClientTestNewProject.kt

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
```

Comment:
Nit: The year in the copyright should be 2021 ;-)

Reply:
... or, if you already started this in 2020, use a range "2020-2021".